files/qsub.pl

```diff
@@ -177,6 +177,7 @@ sub make_command
         'A=s'      => \$account,
         'b=s'      => \$wrap,
         'cwd'      => sub { }, # this is the default
+        'd=s'      => \$workdir,
```

Comment:
you mean like the `wd` option?

Reply:
`qsub -wd` doesn't exist, I don't know why `wd` is even an option

Reply:
in particular, if the `wd` doesn't exist for torque, just rename it? else, use `wd|d=s` to set a 2nd optionname